src/components/ReactTags.tsx

```diff
@@ -41,6 +41,7 @@ type ReactTagsProps = ReactTagsWrapperProps & {
   inputProps: { [key: string]: string };
   editable: boolean;
   clearAll: boolean;
+  accessibilityLabelForClearAll?: string;
```

Comment:
**Type Safety:**
You introduced `accessibilityLabelForClearAll` in the props. Since this is user-facing and can impact the accessibility of the application, consider adding stricter type validation for this prop (e.g., explicitly stating it's a required string).

Reply:
Since  I have given a default value, https://github.com/react-tags/react-tags/blob/e37e8d59b075ec3e50cd60fffcccf2b8b706d152/src/index.tsx#L235 making it a required prop is not necessary I guess.